apps/projects/app/components/Content/Settings.js

```diff
@@ -340,17 +328,23 @@ const BaseRate = ({ baseRate, onChange }) => (
       Define your organization’s hourly rate. This is multiplied by the bounty
       size and converted into the bounty currency under the hood.
     </Text.Block>
-    <Field label="RATE PER HOUR">
+    <FieldTitle style={{marginBottom: '0px'}}>Rate per hour</FieldTitle>
```

Comment:
For consistence and convention zero values are passed without units on styles

Reply:
A-ha. Sure.